gemfiles/.bundle/config

```diff
@@ -0,0 +1 @@
+
```

Comment:
Hmm, git still seems to think this file exists (but with just a blank line). Could you remove it completely?